WordPress/WordPressStatsWidgets/Views/Cards/ListRow.swift

```diff
@@ -0,0 +1,95 @@
+
+import SwiftUI
+import WidgetKit
+
+struct ListRow: View {
+
+    let date: Date
+    let percentValue: Float
+    let value: Int
+
+    private var isToday: Bool {
+        NSCalendar.current.isDateInToday(date)
+    }
+
+    let percentFormatter: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .percent
+        formatter.positivePrefix = "+"
+        return formatter
+    }()
+
+    let dateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.setLocalizedDateFormatFromTemplate("MMM d, yyyy")
+        return formatter
+    }()
+
+    private var differenceBackgroundColor: Color {
+        guard !isToday else {
```

Comment:
I think we might actually want this to be neutral for today or any day in the future – otherwise we get into this situation with sites with timezones ahead of our own:
![IMG_0012](https://user-images.githubusercontent.com/4780/107648391-2c4a3900-6c74-11eb-8702-effdcb8a11b0.PNG)

(In the app, the % and +/- for the 12th would be neutral)